intelmq/tests/bots/experts/uri/test_expert.py

```diff
@@ -0,0 +1,85 @@
+# SPDX-FileCopyrightText: 2022 gutsohnCERT
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# -*- coding: utf-8 -*-
+"""
+Testing uri.
+"""
+
+import unittest
+import intelmq.lib.test as test
+import os
+if os.getenv('INTELMQ_TEST_EXOTIC'):
+    from intelmq.bots.experts.uri.expert import URIExpertBot
+
+EXAMPLE_INPUT = {"__type": "Event",
+                 "source.url": "http://example.com/something/index.php",
+                 "destination.url": "http://example.org/download?file.exe",
+                 "time.observation": "2015-01-01T00:00:00+00:00"
+                 }
+EXAMPLE_OUTPUT = {"__type": "Event",
+                  "source.url": "http://example.com/something/index.php",
+                  "destination.url": "http://example.org/download?file.exe",
+                  "time.observation": "2015-01-01T00:00:00+00:00",
+                  "source.fqdn": "example.com",
+                  "source.scheme": "http",
+                  "source.path": "/something/index.php",
+                  "destination.fqdn": "example.org",
+                  "destination.scheme": "http",
+                  "destination.path": "/download",
+                  "destination.query": "file.exe",
+                  }
+IP_INPUT = {"__type": "Event",
+            "source.url": "http://127.0.0.1/something/index.php",
+            "destination.url": "http://[2001:db8::123]/download?file.exe",
+            "time.observation": "2015-01-01T00:00:00+00:00"
+            }
+IP_OUTPUT = {"__type": "Event",
+             "source.url": "http://127.0.0.1/something/index.php",
+             "source.ip": "127.0.0.1",
+             "destination.url": "http://[2001:db8::123]/download?file.exe",
+             "time.observation": "2015-01-01T00:00:00+00:00",
+             "source.scheme": "http",
+             "source.path": "/something/index.php",
+             "destination.scheme": "http",
+             "destination.path": "/download",
+             "destination.query": "file.exe",
+             }
+
+
+class TestURIExpertBot(test.BotTestCase, unittest.TestCase):
```

Comment:
```suggestion
@test.skip_exotic()
class TestURIExpertBot(test.BotTestCase, unittest.TestCase):
```

Test.skip_exotic() is defined in intelmq/lib/test.py